test/cpp/api/modules.cpp

```diff
@@ -768,6 +768,37 @@ TEST_F(ModulesTest, Linear) {
   }
 }
 
+TEST_F(ModulesTest, LocalResponseNorm) {
+  {
+    LocalResponseNorm model(LocalResponseNormOptions(2));
+    auto x = torch::randn({32, 5, 24, 24}, torch::requires_grad());
+    auto y = model(x);
+    //auto y_exp = torch::layer_norm(x, {2, 2}, model->weight, model->bias, 2e-5);
```

Comment:
For the expected outputs in the module tests, I think we can get the numerical values from Python and directly compare to those hardcoded values :D